rhel7/checks/oval/grub2_enable_fips_mode.xml

```diff
@@ -14,7 +14,7 @@
       <criteria operator="OR">
         <criterion test_ref="test_grub2_enable_fips_mode" comment="check for fips=1 in /etc/default/grub via GRUB_CMDLINE_LINUX" />
         <criteria operator="AND">
-          <criterion test_ref="test_grub2_default_exists" comment="check for GRUB_CMDLINE_LINUX_DEFAULT exists in /etc/default/grub" />
+          <extend_definition definition_ref="test_grub2_default_exists" comment="check for GRUB_CMDLINE_LINUX_DEFAULT exists in /etc/default/grub" />
```

Comment:
I can't find any definition with ID "test_grub2_default_exists" in ssg-rhel7-oval.xml.